keyboards/handwired/atreus50/keymaps/ajp10304/keymap.c

```diff
@@ -145,6 +112,26 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     _______,      _______,      _______,      _______, _______, _______, _______, _______, _______, _______,    _______,    _______,     _______, _______
 ),
 
+
+/* Num Pad
+ * ,------------------------------------------             |-----------------------------------------.
+ * | ESC  |      |      |      |      |      |             |NMLOCK|   7  |   8  |   9  |   /  |      |
+ * |------+------+------+------+------+-------             |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |             |      |   4  |   5  |   6  |   *  |      |
+ * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |             |      |   1  |   2  |   3  |   +  |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |   0  |   .  |   ,  |   -  |      |
+ * `-------------------------------------------------------------------------------------------------'
+ */
+[_NUMPAD] = LAYOUT_ortho_4x12(\
```

Comment:
This keymap doesn't compile because this layout doesn't exist on this board.

```suggestion
[_NUMPAD] = LAYOUT(\
```

Reply:
@noroadsleft Thank you!  Fixed in latest commit.